schema/xsd/portal.xsd

```diff
@@ -59,15 +59,13 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="expiration-time" minOccurs="0" maxOccurs="1" type="xs:dateTime">
+            <xs:element name="available-seconds" minOccurs="0" maxOccurs="1" type="xs:nonNegativeInteger">
```

Comment:
`xs:nonNegativeInteger` is mapped by JAXB to `BigInteger`. Could we jump through some bindings-hoops to have the field in the generated JAXB-class be a `long` instead of `BigInteger`? Since we do `BigInteger.longValue()` on it anyway?